api/src/main/java/org/xwiki/apps/googleapps/internal/CookieAuthenticationPersistenceImpl.java

```diff
@@ -89,38 +77,40 @@
     private static final String EQUAL_SIGN = "=";
     private static final String UNDERSCORE = "_";
 
-    /* Default Cookie TTL : 3 days */
-    private static final COOKIE_TTL = 259200
-
-    private Log logger = LogFactory.getLog(CookieAuthenticationPersistenceStoreTools.class);
+    @Inject
+    private Logger logger;
 
     private XWikiContext context;
-    private XWikiRequest request;
+
+    @Inject
+    private ComponentManager componentManager;
```

Comment:
Where is this used?

Reply:
Removed in 093bd59f4878dffc52fa3fefdf8bfbf14ff9126c where I can now safely rely on having the GoogleAppsManager.